### PR TITLE
Adding the Engineering Addendum to our repository for use by future teams

### PR DESCRIPTION
**Pull request by atjohnson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
\ No newline at end of file
+<div style="text-align: center;">
+
+# **Brain-4ce** - An EEG-based Brain-Computer Interface
+
+</div>
+
+
+<p align="center">
+  <img src="./Documents/brain_4ce_logo.png" alt="Brain 4ce Logo" style="display:block; margin:auto;" width="350">
```
Comment (jonathanm-12) [inline README.md]: Can we decrease the width of the logo to 100-200px?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,67 @@
...
+  <br>
+  <em>Where thoughts become motion.</em>
+</p>
+<!-- <br></br> -->
+<div style="text-align: center;">
+  <h2 style="text-decoration: underline;">Designed by BU ECE 2023 Senior Design Capstone Team 4</h2>
+  <img src="./Documents/team_photo.png" alt="Brain 4ce Logo" style="display:block; margin:auto;" width="600">
```
Comment (jonathanm-12) [inline README.md]: maybe center this photo as well and make it same width?

**Review by jonathanm-12 — COMMENTED** [review]
Added some comments,

Also I think the URLs need to be linked to the main repo not the fork

File: README.md (hunk 1 above)
Comment (atjohnson) [reply README.md]: The logo becomes too small and illegible at <=200px

File: README.md (hunk 2 above)
Comment (atjohnson) [reply README.md]: Keeping the photo as is looks best in my opinion. The centering should be fixed once I push this next commit.

**Comment by atjohnson** [comment]
The URLs are fine

**Review by mitch-gilmore — APPROVED** [review]
Looks great! Thank you @atjohnson for putting this together! You're a rock star!

**Review by jonathanm-12 — APPROVED** [review]

**Comment by jonathanm-12** [comment]
ready to be merged

**Pull request merged**